src/Tooltips/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import ReactTooltip from 'react-tooltip'
+import * as S from './styles'
+import { Props } from "./types"
+
+export const Tooltip = ({ effect, place, tip, children, type }: Props) => {
+    /**
+     * @description the ReactTooltip Component need a different id
+     */
+
+    const id = Math.floor((1 + Math.random()) * 0x10000).toString(16).substring(1) + tip
+    return (
+        <S.Container data-testid="container">
```

Comment:
Temos o componente "coringa" `<Box>` pra evitar a criação desnecessária de containers. Ele já vem tipado com os tokens do design system, caso precise utilizar.

Outro ponto, no geral o atributo customizado `data-testid` não fica com valor fixo na implementação e é passado no arquivo de testes, recomendo tirar onde tem algum valor fixo e caso precise ajustar os seus testes, utilize a query `findByText`.

```suggestion
        <Box css={{ display: 'flex' }} data-testid="container">
```